Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli

```diff
@@ -81,6 +81,7 @@ PixelGeometryData EvaluatePixelGeometry_Eye(VsOutput IN, bool isFrontFace)
 
     float3 vertexNormal, vertexTangent, vertexBitangent;
     ConstructTBN(IN.normal, IN.tangent, objectToWorld, objectToWorldIT, vertexNormal, vertexTangent, vertexBitangent);
+    vertexBitangent = normalize(mul(objectToWorld, float4(IN.bitangent, 0.0)).xyz);
```

Comment:
I think we should have a comment here to explain why we're doing this.

Reply:
Also we should have an Issue to investigate further and remove the bitangent again if possible, or provide an explanation about why the bitangent is necessary.

Reply:
Right. That's what I plan to do. I'll add a comment to explain.

Reply:
https://github.com/o3de/o3de/issues/13159